images/ansible-operator/pipfile.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi8/ubi:8.9-1107 AS basebuilder
+FROM registry.access.redhat.com/ubi9/ubi:9.4-1214 AS basebuilder
```

Comment:
Note: We should update `operator-sdk's` images to the same version as here to not run into issues.